server/src/test/java/org/apache/druid/server/coordinator/DataSourceCompactionConfigTest.java

```diff
@@ -238,7 +234,7 @@ public void testSerdeGranularitySpec() throws IOException
         null,
         new Period(3600),
         null,
-        new UniformGranularitySpec(Granularities.HOUR, null, null),
```

Comment:
Does this change have any impact for upgrades / downgrades?

Reply:
No it does not. The json serialization / deserialization key remains the same